ui/src/actions/sampleChanger.js

```diff
@@ -1,10 +1,18 @@
 /* eslint-disable promise/catch-or-return */
-/* eslint-disable promise/no-nesting */
 /* eslint-disable promise/prefer-await-to-then */
-/* eslint-disable sonarjs/no-duplicate-string */
-import fetch from 'isomorphic-fetch';
 import { showErrorPanel } from './general';
 import { clearCurrentSample } from './queue'; // eslint-disable-line import/no-cycle
+import {
+  fetchLoadedSample,
```

Comment:
There's a naming inconsistency in the back-end between "load" and "mount". I've tried to match the back-end naming - i.e. `fetchLoadedSample` but `sendMountSample` and `sendUnmountSample`.